jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListBasicFiltersPresenter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.pr.client.editors.definition.list;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.jbpm.workbench.common.client.filters.active.ActiveFilterItem;
+import org.jbpm.workbench.common.client.filters.basic.BasicFiltersPresenter;
+import org.jbpm.workbench.pr.client.resources.i18n.Constants;
+import org.uberfire.client.annotations.WorkbenchScreen;
+
+import static org.dashbuilder.dataset.filter.FilterFactory.likeTo;
+import static org.jbpm.workbench.pr.model.ProcessDefinitionDataSetConstants.*;
+
+@ApplicationScoped
+@WorkbenchScreen(identifier = PROCESS_DEFINITION_LIST_BASIC_FILTERS_SCREEN)
+public class ProcessDefinitionListBasicFiltersPresenter extends BasicFiltersPresenter {
+
+    private Constants constants = Constants.INSTANCE;
+
+    @Override
+    public void loadFilters() {
+        view.addTextFilter(constants.Process_Definition_Name(),
```

Comment:
@bxf12315 @cristianonicolai  would make sense create an specific select filter over the existing definitions ones like here?  https://github.com/kiegroup/jbpm-wb/blob/499902e8eeb730d9480cad798a9e93ab4edbcc04/jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListBasicFiltersPresenter.java#L145

Reply:
+1

Reply:
Only suggestion is that the select could be done in a separated PR, with the initial version based on text search.

Reply:
+1

Reply:
o